docs/install/install-node.rst

```diff
@@ -159,11 +159,8 @@ Paste them into a root console on the target node:
 .. task:: install_flocker centos-7
    :prompt: [root@centos]#
 
-Installing ``flocker-node`` will automatically install Docker, but the ``docker`` service may not have been enabled or started.
-To enable and start Docker, run the following commands in a root console:
-
-.. task:: enable_docker centos-7
-   :prompt: [root@centos]#
+Flocker's container management features depend on Docker.
+Make sure `a recent version of Docker (at least 1.8) is installed <https://docs.docker.com/docker/installation/>` and running.
```

Comment:
:question: Remove "recent" - it doesn't add anything if you've already got "at least 1.8" and increases the likelihood of our docs looking outdated if we neglect to update this and leave the word "recent" there for a long time.